src/Aquifer.API/Endpoints/Resources/Content/CreateTranslation/Endpoint.cs

```diff
@@ -20,14 +20,20 @@ public override async Task HandleAsync(Request request, CancellationToken ct)
     {
         var user = await userService.GetUserFromJwtAsync(ct);
         var isCommunityUser = !userService.HasPermission(PermissionName.CreateContent);
+        var response = new Response
+        {
+            ResourceContentId = 0,
+        };
 
         if (isCommunityUser) {
             var isUserAssigned = await dbContext.ResourceContentVersions
                 .AsTracking()
                 .AnyAsync(x => x.AssignedUserId == user.Id, ct);
 
             if (isUserAssigned) {
-                ThrowError("User can only create one translation at a time");
+                response.Message = "User can only create one translation at a time";
```

Comment:
Was there an issue with using the `ThrowError` before? We haven't used this pattern anywhere else and I'd be hesitant to break out of it for this case, especially since FastEndpoints already takes the error from `ThrowError` and formats it for us like:
```
{
	"statusCode": 400,
	"message": "One or more errors occurred!",
	"errors": {
		"generalErrors": ["User can only create one translation at a time"]
	}
}
```

Reply:
As per Jesse, 'we don't need to be throwing errors on that endpoint for trying to start a second translation'.
So, really, the issue on that particular situation is that we don't need that level of verbosity with error reporting - which makes sense. 
I tend to agree as far as following the set paradigm for consistency but I can also see not wanting the error report every time someone clicks translate in a new tab as well.

Reply:
Huh, I guess my mindset is:
1) The error shouldn't ever get alerted on since we're swallowing it on the frontend
2) It _feels_ like it's an actual error. The data is in a different state than the user expects and it's an edge case. Granted, we assume people will regularly run into it, but that's no different from things like filesystem APIs that have regularly expected error types like `FileNotFoundException`.

If we're for sure not ok with making it an error then I wonder if having a boolean of `UserAlreadyAssignedContent` would make simplify things a bit.

Reply:
Totally reasonable. I'll defer to whatever you guys decide on this as I will be ooo until Tuesday.

Reply:
The backend gets the error logged because `ThrowError` does in fact throw an exception to stop things. This doesn't feel like an exceptional situation to me, because it's expected to happen, and while a minor thing, there is more overhead to throwing an exception than just sending back the 400 directly. The caller cares that they were already assigned, and they get that message back. On the back-end I don't think we care.